src/scss/base/_layout.scss

```diff
@@ -45,6 +45,15 @@ body {
         padding-top: spacing(x4);
     }
 }
+
+    // uses flexbox to centre align content both vertically and horizontally
+    .l-content--centred, // British or US English spellings..
```

Comment:
Guessing this would just be `l-centered` as otherwise it implies it would always be used with the `l-content` class as a modifier?